src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -76,7 +76,7 @@ public void FxCopConverter_EndToEnd()
         [Fact]
         public void StaticDriverVerifierConverter_EndToEnd()
         {
-            BatchRunConverter(ToolFormat.StaticDriverVerifier, TestMode.CompareFileContents);
+            BatchRunConverter(ToolFormat.StaticDriverVerifier, "*.tt", TestMode.CompareFileContents);
```

Comment:
Because this was missing, this test never found any test files (because it looks for `.xml` files by default), and so the test was silently succeeding without doing anything.